assets/models/system/stabilityai-stable-diffusion-2-1/spec.yaml

```diff
@@ -3,15 +3,36 @@ name: stabilityai-stable-diffusion-2-1
 path: ./
 properties:
   SHA: 5cae40e6a2745ae2b01ad92ae5043f95f23644d6
-  datasets: LAION-5B
   inference-min-sku-spec: 4|1|28|176
   inference-recommended-sku: Standard_NC6s_v3, Standard_NC12s_v3, Standard_NC24s_v3,
     Standard_NC24rs_v3, Standard_NC16as_T4_v3, Standard_NC24ads_A100_v4, Standard_NC48ads_A100_v4,
     Standard_NC4as_T4_v3, Standard_NC64as_T4_v3, Standard_NC8as_T4_v3, Standard_NC96ads_A100_v4,
     Standard_ND40rs_v2, Standard_ND96amsr_A100_v4, Standard_ND96asr_v4
-  model_id: stabilityai/stable-diffusion-2-1
 tags:
-  Preview: ''
+  Preview: ""
+  SharedComputeCapacityEnabled: ""
   license: creativeml-openrail++-m
   task: text-to-image
-version: 5
+  huggingface_model_id: stabilityai/stable-diffusion-2-1
+  author: stabilityai
+  training_dataset: LAION-5B
+  inference_supported_envs:
+  - ds-mii
+  inference_compute_allow_list:
+    [
+      Standard_NC6s_v3,
+      Standard_NC12s_v3,
+      Standard_NC24s_v3,
+      Standard_NC24rs_v3,
+      Standard_NC16as_T4_v3,
+      Standard_NC24ads_A100_v4,
+      Standard_NC48ads_A100_v4,
+      Standard_NC4as_T4_v3,
+      Standard_NC64as_T4_v3,
+      Standard_NC8as_T4_v3,
+      Standard_NC96ads_A100_v4,
+      Standard_ND40rs_v2,
+      Standard_ND96amsr_A100_v4,
+      Standard_ND96asr_v4
+    ]
+version: 5
```

Comment:
version was not incremented

Reply:
We have model ver 4 in prod registry. We don't need to update version.